Vagrantfile

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
This kind of thing is worth mentioning in the README. Just a brief "by the way you can run the app in a VM if you ..."